src/index.css

```diff
@@ -66,3 +66,132 @@ button:focus-visible {
         background-color: #f9f9f9;
     }
 }
+
+/* Bounding Box css */
+
+.object-detector {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 370px;
+    width: 750px;
+    padding: 20px;
+    background: linear-gradient(360deg, rgba(255, 255, 255, 0.9), rgba(252, 255, 46, 0.97));
+    border-radius: 10px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.image-container {
+    position: fixed;
+    margin-right: 90px;
+    border: 2px solid rgb(204, 204, 204);
+    border-radius: 10px;
+    margin-left: 40px;
+    max-width: 400px;
+    max-height: 230px;
+    overflow: hidden;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
```

Comment:
Let's move the bounding box specific styles to a `BoundBox.css` file to keep everything tidier